aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/helpers/CognitoDeviceHelper.java

```diff
@@ -80,7 +67,7 @@ public static String generateRandomString() {
      * Static class for SRP related calculations for devices.
      */
     @SuppressWarnings("checkstyle:typename")
-    public static class deviceSRP {
+    private static final class deviceSRP {
```

Comment:
Why are we using lowercase for a class?

Reply:
I tried to minimize my impact. I will change this.